### DISCUSSION TITLE
Prefetch on hover disabled in AppRouter?

### DISCUSSION THREAD

**Discussion by Waffe** [vercel/community#4652] [Feature Question]
### Question

So I tried to do as the documentation said, disable prefetching on all links in the navigation, cause loading 15mb of unused data seemed unnecessary.

But according to https://github.com/vercel/next.js/blob/6a8010dbb60606099a22b02a5fe2ab3ddeb5193a/packages/next/src/client/link.tsx#L658-L663

It seems that prefetching on hover is disabled, and theres no option to enable it?
Why?

**Comment by amyegan** [comment]
Hi @Waffe! Link [`prefetch`](https://nextjs.org/docs/app/api-reference/components/link#prefetch) can either be enabled or disabled. I'm not aware of any in-between state to prefetch on hover.

**Reply by Waffe** [reply]
Hello @amyegan! If you currently install NextJS 14 with the page router, it comes with hover functionality enabled. I believe the documentation must have been recently updated. Previously, it mentioned that setting prefetch={false} would only disable the viewport prefetch, while leaving the hover prefetch active.

There seems to be a curiosity among some users about how to completely disable the prefetch feature, including the hover functionality. I can assure you, this observation isn’t merely a figment of imagination! 😄

Referencing the link in the original post, it illustrates the logic behind the hover prefetch. A pertinent question arises: why is this feature disabled for the app router? Consider a scenario where there's an open menu with 12 items; this could prefetch around 5-8mb of data on the initial load. Retaining the hover prefetch feature would preserve the benefits of eager prefetching, enhancing usability speed without compromising loading times and overall user experience before fully engaging with the site.
